README.md

```diff
@@ -90,6 +90,17 @@ Looking for more guidance? Full documentation for Gatsby lives [on the website](
 
 - **To dive straight into code samples, head [to our documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
 
+## We use Yarn!
+[Yarn](https://yarnpkg.com/) is a actively open sourced package manager being developed by Facebook.
+
+* **Offline Mode.** If you've installed a package before, then you can install it again without an internet connection.
```

Comment:
I think we could remove the marketing stuff and just a leave a link for now.

Opening line above is fine after the edit.

I think from here we just want to make sure that yarn is used in all the commands and maybe we add a "yarn install" step to the docs somewhere else.